openstudio-extension.gemspec

```diff
@@ -33,6 +33,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'octokit', '~> 4.18.0' # for change logs
   spec.add_dependency 'openstudio-workflow', '~> 2.1.1'
   spec.add_dependency 'openstudio_measure_tester', '~> 0.2.3'
+  spec.add_dependency 'openstudio-standards', '~> 0.2.12' # for os_lib unit tests
```

Comment:
This wasn't in there before, so why is this needed now?

Reply:
Because the new spec tests exercise methods in the extension gem that use standards. It failed to find standards without this.

Reply:
then perhaps we make this `add_development_dependency`?

Reply:
I think we need to discuss how this is going to work. The extension gem shouldn't have this heavy of a dependency. Can we move all of this into the model articulation gem?